include/amici/edata.h

```diff
@@ -53,19 +53,19 @@ class ExpData {
      * @param observedEventsStdDev (dimension: nmaxevent x nztrue, row-major)
      */
     ExpData(int nytrue, int nztrue, int nmaxevent,
-            std::vector<realtype> ts,
-            std::vector<realtype> observedData,
-            std::vector<realtype> observedDataStdDev,
-            std::vector<realtype> observedEvents,
-            std::vector<realtype> observedEventsStdDev);
+            std::vector<realtype> const& ts,
```

Comment:
I think in most cases constness interferes here with the move inside the setters (works, but move has no effect).